ts/sagas/startup/watchLogoutSaga.ts

```diff
@@ -50,10 +49,6 @@ export function* watchLogoutSaga(
         };
         yield put(logoutFailure(logoutError));
       }
-      // Force the login by expiring the session
-      // FIXME: possibly reset the navigation stack as the watcher of
-      // SESSION_EXPIRED will save the navigation state and later restore it
-      yield put(sessionExpired());
```

Comment:
Removing this line causes the infinite loading in case user wants logout keeping store data